orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -34,37 +36,42 @@ import (
 	mconfig_protos "magma/orc8r/lib/go/protos/mconfig"
 )
 
-var localBuilders = []mconfig.Builder{
-	&baseOrchestratorBuilder{},
+type builderServicer struct {
+	orc8Version string
 }
 
-type builderServicer struct{}
-
-func NewBuilderServicer() builder_protos.MconfigBuilderServer {
-	return &builderServicer{}
+func NewBuilderServicer(orc8rVersion *string) builder_protos.MconfigBuilderServer {
+	if orc8rVersion == nil {
+		orc8rVersion, res := os.LookupEnv("VERSION_TAG")
+		if !res {
+			glog.Error("Couldn't get value for VERSION_TAG Env variable.")
+		}
+		return &builderServicer{orc8rVersion}
+	}
+	return &builderServicer{*orc8rVersion}
 }
 
 func (s *builderServicer) Build(ctx context.Context, request *builder_protos.BuildRequest) (*builder_protos.BuildResponse, error) {
 	ret := &builder_protos.BuildResponse{ConfigsByKey: map[string][]byte{}}
 
-	for _, b := range localBuilders {
-		partialConfig, err := b.Build(request.Network, request.Graph, request.GatewayId)
-		if err != nil {
-			return nil, errors.Wrapf(err, "sub-builder %+v error", b)
-		}
-		for key, config := range partialConfig {
-			_, ok := ret.ConfigsByKey[key]
-			if ok {
-				return nil, fmt.Errorf("builder received partial config for key %v from multiple sub-builders", key)
-			}
-			ret.ConfigsByKey[key] = config
+	config, err := (&baseOrchestratorBuilder{s.orc8Version}).Build(request.Network, request.Graph, request.GatewayId)
+	if err != nil {
+		return nil, errors.Wrapf(err, "builder error")
+	}
+	for key, configToRet := range config {
```

Comment:
Nit: rather than "config" and "configToRet", something like "configs" and "config", respectively, is more descriptive (shows the pluralization)